### ISSUE TITLE
support older SDK install

### ISSUE THREAD

**Issue by axet** [mvdan/fdroidcl#64]
Hello!

fdroidcl does not check if SDK level is supported. app can check maximum avaiable sdk level and install old version of app if available.

* https://f-droid.org/en/packages/com.google.android.stardroid/

Would be nice if fdroidcl selects 'Version 1.9.7 - RC1' for older devices

output of 'fdroidcl show com.google.android.stardroid' also missing required information:

```
Available Versions :

    Version : 1.10.1 - RC1 (1541)
    Size    : 3948966
    MinSdk  : 0

    Version : 1.10.0 - RC3 (1532)
    Size    : 3772159
    MinSdk  : 0

    Version : 1.9.7 - RC1 (1524)
    Size    : 3814707
    MinSdk  : 0
```

**Comment by Linus789** [comment]
Might work now with https://github.com/mvdan/fdroidcl/releases/tag/v0.6.0. If not, reopen the issue.

**Comment by axet** [comment]
Not working.

```
fdroidcl version
v0.6.0
```

```
echo SDK: $(adb shell getprop ro.build.version.sdk)
echo ABI: $(adb shell getprop ro.product.cpu.abi)
SDK: 10
ABI: armeabi-v7a
```

```
fdroidcl show com.github.axet.audiorecorder

    Version : 3.5.15 (371)
    Size    : 5971283
    MinSdk  : 9
    ABIs    : arm64-v8a, armeabi, armeabi-v7a, x86, x86_64

    Version : 3.5.13 (369)
    Size    : 5971282
    MinSdk  : 9
    ABIs    : arm64-v8a, armeabi, armeabi-v7a, x86, x86_64


    Version : 3.5.12 (368)
    Size    : 5967184
    MinSdk  : 9
    ABIs    : arm64-v8a, armeabi, armeabi-v7a, x86, x86_64
```

**Comment by Linus789** [comment]
What is the output of
```
$ adb shell getprop | grep 'ro\.product\.cpu\.abi'
$ adb shell getprop ro.product.cpu.abilist
$ adb shell getprop ro.product.cpu.abi
$ adb shell getprop ro.product.cpu.abi2
```
?

**Comment by axet** [comment]
```
# adb shell getprop | grep 'ro\.product\.cpu\.abi'
[ro.product.cpu.abi]: [armeabi-v7a]
[ro.product.cpu.abi2]: [armeabi]
# adb shell getprop ro.product.cpu.abilist

# adb shell getprop ro.product.cpu.abi
armeabi-v7a
#  adb shell getprop ro.product.cpu.abi2
armeabi
```

**Comment by Linus789** [comment]
Should be fixed with https://github.com/mvdan/fdroidcl/commit/ad4e8090459410bf9e6b38fbd7c6fbf6b1c07716.

**Comment by axet** [comment]
Will strings.TrimSpace(result) do the trick?

**Comment by Linus789** [comment]
Yes, that would work as well. In the end I opted for stripping one optional newline at the end which I did in a later commit.

**Comment by Linus789** [comment]
Should be fixed in [v0.7.0](https://github.com/mvdan/fdroidcl/releases/tag/v0.7.0).
